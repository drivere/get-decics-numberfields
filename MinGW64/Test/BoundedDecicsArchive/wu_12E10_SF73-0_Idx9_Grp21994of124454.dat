[y^2 - 73, [2, 0], 73, 2, [[1, -4.7720018726587655839358241631198532173; 1, 3.7720018726587655839358241631198532173], [1, -4.7720018726587655839358241631198532173; 1, 3.7720018726587655839358241631198532173], [1, -5; 1, 4], [2, -1; -1, 37], [73, 37; 0, 1], [37, 1; 1, 2], [73, [37, 18; 1, 36]]], [-8.5440037453175311678716483262397064346, 8.5440037453175311678716483262397064346], [1, 1/2*y - 1/2], [1, 1; 0, 2], [1, 0, 0, 18; 0, 1, 1, -1]]
0
9
1
-49  -10
-2
-2
42
42
23
44
