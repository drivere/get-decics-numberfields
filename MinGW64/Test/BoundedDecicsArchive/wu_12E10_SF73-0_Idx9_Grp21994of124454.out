#   The search is complete. Stats:
#   Inspected 9570 polynomials.
#   Num Polys passing irreducibility test = 9570 (100%).
#   Num Polys passing discriminant bound = 0 (0%).
#   Elapsed Time = 132 (sec)
